client/app/components/Link.tsx

```diff
@@ -0,0 +1,22 @@
+import React
+ from "react";
+interface LinkProps {
+    text: string;
+    title: string;
+    link: string;
+    isNewTab?: boolean;
+}
+
+export default function Link({text, title, link, isNewTab =false}: LinkProps) {
+    return (
+        <a 
+            href={link}
+            title={title}
+            target={isNewTab ? '_blank' : '_self'}
+            rel={isNewTab ? 'noopener noreferrer' : undefined}
+            className="text-blue-500 hover:underline"
```

Comment:
spatiere

```suggestion
            href = {link}
            title = {title}
            target = {isNewTab ? '_blank' : '_self'}
            rel = {isNewTab ? 'noopener noreferrer' : undefined}
            className = "text-blue-500 hover:underline"
```

Also la rel nu sunt sigur ca e ok asa. Daca `isNewTab` e false s-ar putea sa iti rama in componenta 'rel="undefined"' ceea ce nu e ok. Ma documentez si revin